cmd/server/cmd/server.go

```diff
@@ -218,6 +227,27 @@ func runServer(sa *serverArgs, repo template.Repository, adapters []pkgAdapter.I
 		}
 		eval = ilEval
 	}
+
+	var dispatcher mixerRuntime.Dispatcher
+
+	// TODO until the dispatcher 2 switch is complete,
+	// dispatcher 2 is only enabled when configStore2URL is specified.
+	if sa.configStore2URL != "" {
+		adapterMap := adapter.InventoryMap(adapters)
+		store2, err := store.NewRegistry2(config.Store2Inventory()...).NewStore2(sa.configStore2URL)
+		if err != nil {
+			fatalf("Failed to connect to the configuration2 server. %v", err)
```

Comment:
to the configuration server

Reply:
we will replace all "*2". Otherwise messages for 1 and 2 both look the same.